__tests__/app.test.js

```diff
@@ -131,6 +154,47 @@ describe("GET /api/articles", () => {
         expect(articles).toBeSorted({ key: "created_at", descending: true });
       });
   });
+  test("STATUS 200: returned array of articles is filtered by topic query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+
+        expect(articles.length).toBe(12);
+      });
+  });
+  test("STATUS 200: returned array of articles is filtered by topic query sorted by created_at by default", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+
+        expect(articles.length).toBe(12);
```

Comment:
you don't need to test this again as you already have a test for the length prior to this test

Reply:
noted!